packages/boxplot/src/BoxPlotItem.tsx

```diff
@@ -0,0 +1,242 @@
+import { createElement, MouseEvent, useCallback, useMemo } from 'react'
+import { animated, SpringValue, to } from '@react-spring/web'
+import { useTooltip } from '@nivo/tooltip'
+import { BoxPlotDatum, BoxPlotItemProps, BoxPlotSummary } from './types'
+
+const BoxPlotItemWhiskers = ({
+    bandwidth,
+    coordinates,
+    layout,
+    whiskerColor,
+    whiskerWidth,
+    whiskerEndWidth,
+}: {
+    bandwidth: number
+    coordinates: Pick<BoxPlotSummary, 'values'>
+    layout: 'vertical' | 'horizontal'
+    whiskerWidth: number
+    whiskerColor: string | SpringValue<string>
+    whiskerEndWidth: number
+}) => {
+    const whiskerEndGap = ((1 - whiskerEndWidth) * bandwidth) / 2
+    if (layout === 'vertical') {
+        const y = coordinates.values[3]
+        return (
+            <>
+                <animated.line
+                    x1={to(bandwidth / 2, value => Math.max(value, 0))}
```

Comment:
The new approach does use a rotation.

On this topic but slightly different - the animation now starts at the median line and expands the boxes outward. What do you think of that? It could also go more side-to-side.

Reply:
I think it's nice like this.